Bigtable/src/RowMutation.php

```diff
@@ -0,0 +1,150 @@
+
+    public function __construct($rowKey, array $options = [])
+    {
+        $this->rowKey = $rowKey;
+    }
+
+    /**
+     * Return the row key.
+     *
+     * @return string RowKey of the Row.
+     */
+    public function getRowKey()
+    {
+        return $this->rowKey;
+    }
+
+    /**
+     * Creates Insert/Update mutation for a row.
+     *
+     * @param string $family Family name of the row.
+     * @param string $qualifier Column qualifier of the row.
+     * @param string $value Value of the Column qualifier.
+     * @param int $timeStamp optional timestamp value.
+     *
+     * @return void
+     */
+    public function upsert($family, $qualifier, $value, $timeStamp = 0)
+    {
+        $mutation = new Mutation;
+        $mutationSetCell = new Mutation_SetCell;
+        $mutationSetCell->setFamilyName($family)->setColumnQualifier($qualifier)
+                        ->setValue($value);
+        if ($timeStamp === 0) {
+            $mutationSetCell->setTimestampMicros(time() * 1000);
+        } else {
+            $mutationSetCell->setTimestampMicros($timeStamp);
+        }
+        $mutation->setSetCell($mutationSetCell);
+        $this->mutations[] = $mutation;
+    }
+
+    /**
+     * Creates delete from family mutation for a row.
+     *
+     * @param string $family Family name of the row.
+     *
+     * @return void
+     */
+    public function deleteFromFamily($family)
+    {
+        $mutation = new Mutation;
+        $deleteFromFamily = new Mutation_DeleteFromFamily;
+        $deleteFromFamily->setFamilyName($family);
+        $mutation->setDeleteFromFamily($deleteFromFamily);
+        $this->mutations[] = $mutation;
+    }
+
+    /**
+     * Creates delete from column mutation for a row.
+     *
+     * @param string $family Family name of the row.
+     * @param string $qualifier Column qualifier of the row.
+     * @param array $timeRange optional array of time range to delete from column,
+     *        keyed by `start` and `end` representing time range window.
+     * @return void
+     */
+    public function deleteFromColumn($family, $qualifier, array $timeRange = [])
+    {
+        $mutation = new Mutation;
+        $deleteFromColumn = new Mutation_DeleteFromColumn;
+        $deleteFromColumn->setFamilyName($family)->setColumnQualifier($qualifier);
+        if (!empty($timeRange)) {
+            $timestampRange = new TimestampRange;
+            $timestampRange->setStartTimestampMicros($timeRange['start']);
+            $timestampRange->setEndTimestampMicros($timeRange['end']);
+            $deleteFromColumn->setTimeRange($timestampRange);
+        }
+        $mutation->setDeleteFromColumn($deleteFromColumn);
+        $this->mutations[] = $mutation;
+    }
+
+    /**
+     * Creates delete row mutation for a row.
+     *
+     * @return void
+     */
+    public function deleteFromRow()
+    {
+        $mutation = new Mutation;
+        $mutation->setDeleteFromRow(new Mutation_DeleteFromRow);
```

Comment:
In the latest version of protoc, I think Mutation_DeleteFromRow is now generated as Mutation\DeleteFromRow now.
Do you know who will update that? @ajaaym

Reply:
We just need to re-run the synth tool (using [this script](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Bigtable/synth.py)) to regenerate the GAPICs and protos. We are planning to do this and clean up any namespaces prior to the release.